### ISSUE TITLE
Difficult to select network objects

### ISSUE THREAD

**Issue by MartijnHarmenzon** [eclipse/sumo#4080]
On my computer, Windows 10 Home with screen resolution 2560x1440, it is very difficult to select network objects like junction, detectors, etc. The underlaying edge is always being selected. My solution now is to switch on the 'object identification tool'. Than the name of the object is being visualized. When it shows the name of a junction I can click on the junction. But this is not a good solution in my opinion.

**Comment by namdre** [comment]
it would be good to narrow down the conditions for that error as it does not always show up.
- does it occur in all networks or only some?
- does the error depend on the screen resolution?
- do you have multiple computers where this shows up? (If so, what operating system are they using)

**Comment by MartijnHarmenzon** [comment]
It occurs in all networks. It does not depend on the sreen resolution. I have not tried any other computer, but I will when I have the chance. 
The issue seems simular to being reported here:
https://github.com/eclipse/sumo/issues/4077#issuecomment-389554576

**Comment by namdre** [comment]
Yes. I suspect it is related to windows 10 (which nobody is using yet in our department).

**Comment by palvarezlopez** [comment]
@MartijnHarmenzon Thanks for the Reporting. I'm going to test in my Windows 10 Computer: Two questions:

1 - Are you using the GIT version of SUMO, or a official release?
2 - If is the GIT version, are yoy used the SUMOLibraries to compile it?

**Comment by MartijnHarmenzon** [comment]
I tested it on another Windows 10 computer just now with a fresh install of the latest nightly build. And it works fine on this pc. The difference between this pc and my pc is Windows 10 Pro vs Windows 10 Home. On my pc with Windows 10 Home it does not work.

**Comment by namdre** [comment]
As object selection depends strongly on OpenGL functions, this might be due to differences in the graphics hardware or OpenGL driver version. 
Still, there may be a workaround possible on the sumo side. All we need is a computer under our control that can reproduce the issue.

**Comment by MartijnHarmenzon** [comment]
@namdre I managed to resolve the problem.

You were right. The problem was related to the graphics on my pc. My pc has Intel HD 520 graphics. While the other pc has has Nvidia graphics. 

First I tried to update my Intel HD 520 graphics driver. Without a result. But after some internet search (Google), I found out that there was another mayor driver update available for the Windows 10 April 2018 update. So after installing the Windows update and again an Intell driver update, SUMO netedit works fine with regards to the object selection.

I hope this might helps others in the future.

**Comment by namdre** [comment]
Thank you for letting us know!

**Comment by xunliusimon** [comment]
I just met this problem with win10 and SUMO 1.7, Graphic Interl 620. I have updated my driver but the problem still exist. Has anyone else reported about this question?

**Comment by namdre** [comment]
Yes. The issues does come up rather often: https://sumo.dlr.de/docs/FAQ.html#distorted_view_with_green_lines_all_over_the_network
